prescripted_countries/!!_hfe_prescripted_discovery.txt

```diff
@@ -6,39 +6,45 @@ discovery = {
 	spawn_enabled = no
 	spawn_as_fallen = no
 	ignore_portrait_duplication = yes
-	ship_prefix = ""
+	ship_prefix = "Discovery"
```

Comment:
no prefix

Reply:
Why?

Reply:
Also, notice that this one is marked as outdated.